cli/command/container/stats.go

```diff
@@ -199,12 +207,13 @@ func runStats(dockerCli *command.DockerCli, opts *statsOptions) error {
 	var err error
 	for range time.Tick(500 * time.Millisecond) {
 		cleanHeader()
-		cStats.mu.RLock()
-		csLen := len(cStats.cs)
-		if err = formatter.ContainerStatsWrite(statsCtx, cStats.cs); err != nil {
+		cStats.mutex.RLock()
+		csList := cStats.copyContainersStats()
```

Comment:
I hate the idea of copying the containers statistics (it won't scale nicely).
I am working on a better way to avoid this part.